### ISSUE TITLE
plot.ceg

### ISSUE THREAD

**Issue by FedericoCarli** [gherardovarando/stagedtrees#58]
We have also to implement the plot function also for the ceg.

**Comment by gherardovarando** [comment]
I have some concerns on this: 

- I would prefer not to add `igraph` as dependency just to plot `ceg`; considering also that the plot with `igraph` is not perfect.
- It would be nice to have our function to plot `ceg` similar to `plot.sevt`
- I have no idea now how to implement `plot.ceg`

Probably we could just show (in a vignette?) how to plot `ceg` using `igraph` :
```
library(stagedtrees)
library(magrittr)
library(igraph)

data("PhDArticles")
model <- PhDArticles %>%  full(lambda = 1) %>%  naive.sevt

model %>% ceg.sevt %>% ceg2adjmat %>% 
  graph_from_adjacency_matrix %>% plot(layout = layout_as_tree) 
```

**Comment by gherardovarando** [comment]
We should implement `plot.ceg` without using external library. 
`plot.ceg` should:
- collapse useless part of the tree
- being able as `plot.sevt` to ignore given stages 
- probably `ceg.sevt` should also be able to ignore some stages (e.g. `NA`)

**Comment by gherardovarando** [comment]
an implementation of `plot.ceg` using the igraph package is now available in #90 and will be available in `v2.2.0` .
